docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx

```diff
@@ -0,0 +1,166 @@
+---
+id: rak7201
+sidebar_label: RAK7201 WisNode Button 4K
+title: RAK7201 WisNode Button 4K
+description: The RAK7201 WisNode Button 4K is a remote wireless trigger device that supports user-defined functions for each key and is based on the LoRaWAN 1.0.2 protocol.
+slug: /use-the-network/devices/ready-to-use/rakwireless/rak7201
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/ready-to-use/rakwireless/rak7201/RAK7201.png"
+  )}
+/>
+
+
+## Overview
+
+The RAK7201 WisNode Button 4K is a remote wireless trigger device. It supports user-defined functions for each key and is based on the LoRaWAN 1.0.2 protocol. Supported LoRaWAN bands are IN865, EU868, AU915, US915, KR920, and AS923.
+
+This WisNode Button is suitable for a variety of Smart Home applications: entertainment system control, control of lights, a snooze button for the alarms, or a remote trigger. With long-range wireless connectivity, amazing battery life, and four programmable buttons, there are a ton of applications this device can fit into.
```

Comment:
Hi @kneelyrac I suggest we change this paragraph a bit as its a bit hard to understand. I like using the https://hemingwayapp.com/ to help me come up with more bold and clear sentences :)

I attached a screenshot of what the hemminwayapp was saying.

<img width="1512" alt="Screen Shot 2022-06-13 at 11 35 44 AM" src="https://user-images.githubusercontent.com/19807431/173391107-545558da-4398-4b54-9431-459132bb5740.png">


Below I think I was able to improve it a bit. Let me know what you think.

Reply:
Thank you for your suggestion, @Perronef5. :)